modules/administration-guide/partials/assembly_enabling-readiness-init-containers.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-assembly_enabling-readiness-init-containers: {context}
+
+[id="assembly_enabling-readiness-init-containers_{context}"]
+= Enabling readiness init containers
+
+:context: assembly_enabling-startup-init-containers
+
+{prod-short} Operator installs {prod-short} and starts its containers in the right order. However, suppose a node with {prod-short} gets restarted. In that case, all {prod-short} containers start simultaneously, causing some containers to fail because of the dependent component that might not be ready yet. To handle such a case, enable readiness init containers for {prod-short} components.
```

Comment:
```suggestion
{prod-short} Operator installs {prod-short} and starts its containers in the right order. However, suppose a node with {prod-short} gets restarted. In that case, all {prod-short} containers start simultaneously, causing some containers to fail because of the dependent component that might not be ready yet. To handle such a case, enable queue init containers for {prod-short} components.
```